run-compose.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
This should be ` #!/bin/env bash` to rely upon env to get the real location of bash